test/powershell/Modules/Microsoft.PowerShell.Utility/Get-Date.Tests.ps1

```diff
@@ -37,6 +37,14 @@ Describe "Get-Date DRT Unit Tests" -Tags "CI" {
         Get-date -Date 1/1/0030 -uformat %S%T%u%U%V%w%W%x%X%y%Y%% | Should be "0000:00:002012001/01/3000:00:00300030%"
     }
 
+    It "Passing <name> to -uformat produces a descriptive error" -TestCases @(
```

Comment:
I would suggest putting single quotes around it to make it more readable in the test output:

```none
It "Passing '<name>' to ...
```

Reply:
@SteveL-MSFT Done